src/sfctl/custom_compose.py

```diff
@@ -64,18 +64,18 @@ def create_app_health_policy(
     )
 
 
-def create(client, name, file_path, user=None, has_pass=False, #pylint: disable=missing-docstring
+def create(client, deployment_name, file_path, user=None, has_pass=False, #pylint: disable=missing-docstring
```

Comment:
One more comment, you can optionally add yourself to the root `CODEOWNERS` file so you will have ownership of the compose files. If you do, be sure to add yourself for the help, test and code files.